test/UnitTests/PlatformServices.Universal.Unit.Tests/packages.config

```diff
@@ -2,5 +2,6 @@
 <packages>
   <package id="Castle.Core" version="4.0.0" targetFramework="net452" />
   <package id="Moq" version="4.7.8" targetFramework="net452" />
+  <package id="Microsoft.Internal.TestPlatform.ObjectModel" version="14.0.0" targetFramework="net452" />
```

Comment:
We can take only this change though.

Reply:
I make only this change on the #310. I still want to find a way how it will work on my PC and without potential problems for you CI setup. If there any hints what bad could happens on CI, I maybe could change something.

Reply:
We are still trying to investigate this. We will get back to you once we figure his out. We will keep this PR open to track this issue. Meanwhile, will push in the other PR.

Reply:
@kant2002 : There is a slight change in plan. We will close this PR from here. I have created a separate issue to track this action item here : https://github.com/Microsoft/testfx/issues/311